### PR TITLE
Testcase page (fixes #68)

### LINKED ISSUE #68: Permalink to test case
Show test case on `testcases/{uid}`

### PR DESCRIPTION
**Pull request by just-boris**
Fixes #68
image

### CODE REVIEW

**Comment by baev** [comment]
screen shot 2017-02-03 at 01 42 56

**Comment by eroshenkoam** [comment]
retest this please

**Comment by baev** [comment]
@just-boris is that ready to merge?

**Comment by just-boris** [comment]
No, it is not.
I still need to fix "close" and "maximize" button on testcase panel.
Will be done in a next few days

File: allure-web/src/layouts/testcase/TestcaseLayout.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export default class TestcaseLayout extends PaneLayout {
+    loadData() {
+        return Promise.resolve();
+    }
+
+    onStateChange() {
+        const changed = Object.assign({}, this.state.changed);
+        const paneView = this.getChildView('content');
+        if(router.previousUrl && !router.previousUrl.startsWith('testcase') && !paneView.getRegion('back')) {
```
Comment (just-boris) [inline allure-web/src/layouts/testcase/TestcaseLayout.js]: This workaround with `startsWith` is terrible.

Maybe we should consider an approach wth query param. We can write a helper `{{testcase-link uid=uid}}` that will render a link to a test case based on current url.

**Comment by just-boris** [comment]
Aside from my comment, the rest is working fine.
Please check it and merge, if it is okay for you.

In next episodes, I will continue the process of making Testcase page great.

File: allure-web/src/layouts/testcase/TestcaseLayout.js (hunk 1 above)
Comment (baev) [inline allure-web/src/layouts/testcase/TestcaseLayout.js]: thats fine...

**Comment by baev** [comment]
@just-boris thanks!

**Pull request merged**